tests/test_single_point.py

```diff
@@ -80,8 +80,8 @@ def test_potential_energy(struct, expected, properties, prop_key, calc_kwargs, i
             "toluene.xyz",
             {"model_path": NEQUIP_PATH},
         ),
-        ("orb", "cpu", -27.088973999023438, "NaCl.cif", {}),
-        ("orb", "cpu", -27.088973999023438, "NaCl.cif", {"model_path": "orb-v2"}),
+        ("orb", "cpu", -27.08186149597168, "NaCl.cif", {}),
+        ("orb", "cpu", -27.089094161987305, "NaCl.cif", {"model_path": "orb-v2"}),
```

Comment:
Do we need them to this precision if comparison is rel=1e-5?, just do 6d.p.

Reply:
As a test which needs to be consistent across different systems, I don't think we can guarantee better precision than somewhere around rel=1e-5, but occasionally it is useful to compare to a higher level of precision for a specific test, so I don't think it hurts to have essentially a note of what I think it should be?